hamilton/execution/executors.py

```diff
@@ -172,9 +206,7 @@ def submit_task(self, task: TaskImplementation) -> TaskFuture:
         :param task: Task to submit
         :return: Future associated with this task
         """
-        # No error management for now
-        result = base_execute_task(task)
-        return TaskFuture(get_state=lambda: TaskState.SUCCESSFUL, get_result=lambda: result)
+        return TaskFutureWrappingFunction(functools.partial(base_execute_task, task))
```

Comment:
Does this add error management?

Reply:
I believe that I removed that comment prematurely. There is no direct error management in `TaskFutureWrappingFunction`. However, there is now error management around the `post_task_resolution` hook in `run_graph_to_completion` (it gets sent to the hook before propagating upward). I would be happy to add the comment back if you like!